compiled_starters/javascript/README.md

```diff
@@ -0,0 +1,32 @@
+![progress-banner](https://codecrafters.io/landing/images/default_progress_banners/shell.png)
+
+This is a starting point for JavaScript solutions to the
+["Build Your Own Shell" Challenge](https://app.codecrafters.io/courses/shell/overview).
+
+_Add a description of your course here_
```

Comment:
@glokta1 let's pay more attention and thoroughly read a PR diff before opening for review. Let's fix this - copy description from https://app.codecrafters.io/courses/shell/overview.